translations/translation_utility.sh

```diff
@@ -3,7 +3,7 @@
 echo -ne "
     ╔══════════════════════════════════════╗
     ║                                      ║
-    ║ Linux/MacOS Translation utility 1.0  ║
+    ║     macOS Translation utility 1.0    ║
```

Comment:
I'd rather leave that.
It'd still work for Linux, with a proper `bin_path` provided by the user